stonesoup/sensormanager/base.py

```diff
@@ -34,6 +41,24 @@ class SensorManager(Base, ABC):
                           "vector in the case of multi-objective optimisation. Metrics may be of "
                           "any type and in any units.")
 
+    take_sensors_from_platforms: bool = Property(
```

Comment:
Quite a long variable name, could we call in "platform_sensors" and explain it's functionality in the doc string?